tests/test_dtel.py

```diff
@@ -0,0 +1,289 @@
+from swsscommon import swsscommon
+import time
+import re
+import json
+
+class TestDtel(object):
+    def test_DtelGlobalAttribs(self, dvs):
+    
+        db = swsscommon.DBConnector(4, dvs.redis_sock, 0)
+        adb = swsscommon.DBConnector(1, dvs.redis_sock, 0)
+    
+        # create DTel global attributes in config db
+        tbl = swsscommon.Table(db, "DTEL")
+
+        fvs = swsscommon.FieldValuePairs([("SWITCH_ID", "1")])
+        tbl.set("SWITCH_ID", fvs)
+
+        fvs = swsscommon.FieldValuePairs([("FLOW_STATE_CLEAR_CYCLE", "10")])
+        tbl.set("FLOW_STATE_CLEAR_CYCLE", fvs)
+
+        fvs = swsscommon.FieldValuePairs([("LATENCY_SENSITIVITY", "100")])
+        tbl.set("LATENCY_SENSITIVITY", fvs)
+
+        fvs = swsscommon.FieldValuePairs([("INT_ENDPOINT", "TRUE")])
+        tbl.set("INT_ENDPOINT", fvs)
+
+        fvs = swsscommon.FieldValuePairs([("INT_TRANSIT", "TRUE")])
+        tbl.set("INT_TRANSIT", fvs)
+
+        fvs = swsscommon.FieldValuePairs([("POSTCARD", "TRUE")])
+        tbl.set("POSTCARD", fvs)
+
+        fvs = swsscommon.FieldValuePairs([("DROP_REPORT", "TRUE")])
+        tbl.set("DROP_REPORT", fvs)
+
+        fvs = swsscommon.FieldValuePairs([("QUEUE_REPORT", "TRUE")])
+        tbl.set("QUEUE_REPORT", fvs)
+
+        fvs = swsscommon.FieldValuePairs([("Ethernet0", "Ethernet0"), ("Ethernet4", "Ethernet4")])  
+        tbl.set("SINK_PORT_LIST", fvs)
+
+        fvs = swsscommon.FieldValuePairs([("INT_L4_DSCP_VALUE", "128"), ("INT_L4_DSCP_MASK", "255")])  
+        tbl.set("INT_L4_DSCP", fvs)
+
+        time.sleep(1)
+
+        atbl = swsscommon.Table(adb, "ASIC_STATE:SAI_OBJECT_TYPE_DTEL")
+        keys = atbl.getKeys()
+        
+        for k in keys:
```

Comment:
i see, here the keys is null then, no assert is checked, that is why it is passing.

Reply:
Yes

Reply:
need to assert len(keys) > 1

Reply:
The check would fail if platform is not barefoot. Is there a "vs platform" check that we can add to orchdaemon.cpp to initialize dtel, in addition to barefoot?